### PR TITLE
i18n update blog post

### PR DESCRIPTION
**Pull request by vanessayuenn**
cc @zeke for collaboration 👀

### CODE REVIEW

**Comment by zeke** [comment]
Love it!

I will make a few tweaks and add some content about the Node.js project.

Here's a PR for the updated contribution guidelines for translators: https://github.com/electron/electron-i18n/pull/249

**Comment by vanessayuenn** [comment]
👀 *pokes @zeke* any update on this?

**Comment by zeke** [comment]
Howdy! Let's ship it this week. I'll put it on my list.

**Comment by vanessayuenn** [comment]
@zeke, I took the liberty to update some stats in the post and finish off the WIP sections. This blog post should be good to go now!  🙌 I think we should publish this ASAP, so there is a bit of a break before the [next blog post](https://github.com/electron/electronjs.org/pull/1283) goes out.

File: data/blog/i18n-updates.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<figure>
+  <img class="screenshot" src="https://user-images.githubusercontent.com/6842965/36511386-c32e31fc-1766-11e8-8484-7466be6a5eb0.png" alt="New Electron documentation footer in Japanese">
+  <figcaption>Electron documentation footer in Japanese</figcaption>
+</figure>
+
+Notice a typo or an incorrect translation while you're reading the documentation? You no longer have to log in to Crowdin, pick your locale, find the file you'd like the fix, etc etc. Instead, you can just scroll down to the bottom of the said doc, and click "Translate this doc" (or the equivalent in your language). Voila! You are brought straight to the Crowdin translation page. Now apply your translation magic!
+
+## 📈 Some Statistics
+
+Ever since we have publicized the Electron documentation i18n effort, we have seen _huge_ growth in translation contributions from Electron community members from all around the world. To date, we have **1,719,029 string translated, from 1,066 community translators, and in 25 languages**.
```
Comment (50Wliu) [inline data/blog/i18n-updates.md]: string -> strings

File: data/blog/i18n-updates.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+If you are one of our incredible translators, please take a few minutes to fill this out: https://goo.gl/forms/b46sjdcHmlpV0GKT2.
+
+## 🙌 Node's Internationalization Effort
+
+Because of the sucess of Electron's i18n initiative, Node.js decided to model [their revamped i18n effort](https://github.com/nodejs/i18n) after the pattern we use as well! 🎉 The [Node.js i18n initiative](https://github.com/nodejs/i18n) has now been launched and gained great momentum, but you can stil read about the early proposal and reasoning behind it [here](https://medium.com/the-node-js-collection/internationalizing-node-js-fe7761798b0a).
+
+## 🔦 Contributing Guide
+
+If you're interested in joining our effort to make Electron more international friendly, we have a hand-dandy [contributing guide](https://github.com/electron/i18n/blob/master/contributing.md) to help you get started. Happy internationalizing! 📚
```
Comment (zeke) [inline data/blog/i18n-updates.md]: hand-dandy -> handy-dandy

👐

**Pull request merged**